attackers/response_handlers.py

```diff
@@ -0,0 +1,23 @@
+from PIL import Image
+import datetime
+import os
+
+
+def string_handler(params):
+    print(f"Server response:\n\t{params[0]}")
+    return params[1:]
+
+
+def image_handler(params: Image.Image):
+    time = datetime.datetime.now().strftime("%Y-%m-%d %H-%M-%S")
+    desktop = os.path.join(os.path.expanduser("~/OneDrive"), "Desktop")
+    screenshots_directory = os.path.join(desktop, "Server_ScreenShots")
+    try:
+        os.mkdir(screenshots_directory)
+    except FileExistsError:

```

Comment:
I really dont like the aproch of beigng sorry instead of asking for permission (tho it is an on one).   
instead of catching `FileExistsError` check if it exists first.  
Any way you could use `makedirs(..., exist_ok=True)` or something like that